src/NServiceBus.Persistence.DynamoDB/Testing/TestableDynamoDBSynchronizedStorageSession.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus.Testing
+{
+    using Extensibility;
+    using Persistence;
+    using Persistence.DynamoDB;
+
+    /// <summary>
+    /// A fake implementation for <see cref="SynchronizedStorageSession"/> for testing purposes.
+    /// </summary>
+    public class TestableDynamoDBSynchronizedStorageSession : ISynchronizedStorageSession
```

Comment:
Does this also need to implement `IDynamoDBStorageSession` to play nicely with the `SynchronizedStorageSessionExtensions` in tests, or do we expect users to implement their `TestableDynamoDBSynchronizedStorageSession`, or am I missing something? (which is most likely 😜)